MAINTAINERS.md

```diff
@@ -8,5 +8,10 @@ Note that the documentation for the Parca Agent is located in the [parca.dev web
 
 Refer to the following maintainers with their focus areas:
 
-* Kemal Akkoyun <https://github.com/kakkoyun> @kakkoyun: Everything (eBPF, profiler, symbolization).
-* Sumera Priyadarsini <https://github.com/Sylfrena> @Sylfrena: Debuginfod integration, Service Discovery and end-to-end tests.
+* Frederic Branczyk <https://github.com/brancz> @brancz: Everything (eBPF, profiler, debuginfo, symbolisation)
+* Kemal Akkoyun <https://github.com/kakkoyun> @kakkoyun: Everything (eBPF, profiler, CI/CD)
```

Comment:
I think it would be appropriate to also put the debuginfo part into the parenthesis. Kemal is much more familiar with the extraction part, whereas I'm more familiar with the upload protocol.